examples/full-nixos/arion-compose.nix

```diff
@@ -0,0 +1,11 @@
+{
+  docker-compose.services.webserver = { pkgs, ... }: {
+    nixos.useInit = true;
+    nixos.configuration.services.nginx.enable = true;
+    nixos.configuration.services.nginx.virtualHosts.localhost.root = "${pkgs.nix.doc}/share/doc/nix/manual";
+    service.useHostStore = true;
```

Comment:
Why is this needed? Maybe it should be turned on by default?

Reply:
It's not strictly needed, but it is currently the only way to do Nix-based images :)
The alternative will be to integrate dockerTools, #6